server/data/tokenStore.ts

```diff
@@ -1,29 +1,29 @@
-import redis from 'redis'
+import { createClient } from 'redis'
 import { promisify } from 'util'
 
 import logger from '../../logger'
 import config from '../config'
 
-const createRedisClient = () => {
-  return redis.createClient({
+export type RedisClient = ReturnType<typeof createClient>
+
+export const createRedisClient = (): RedisClient => {
```

Comment:
In the template the redis stuff is in a `redisClient` file rather than been in here..